docs/content/doc/installation/from-package.en-us.md

```diff
@@ -47,9 +47,52 @@ pacman -S gitea
 
 There is a [Gitea Snap](https://snapcraft.io/gitea) package which follows the latest stable version.
 
-``sh
-snap install gitea
-``
+### This guide was tested with Ubuntu 20.04 and MySQL:
+
+* Install needed Packages and the snap:
+
+```
+sudo snap install gitea
+sudo apt install nginx mariadb-server
+```
+* Secure your mysql Installation
+
+``sudo mysql_secure_installation``
+
+* Create your Database and Database User:
+
+```
+sudo mysql -u root -p
+        CREATE DATABASE gitea;
+        GRANT ALL PRIVILEGES ON gitea.* TO 'gitea'@'localhost' IDENTIFIED BY "StrongPassword";
+        FLUSH PRIVILEGES;
+        QUIT;
+```
+
+* Create nginx Config to pass traffic to port 3000:
+ 
+ ``sudo nano /etc/nginx/conf.d/gitea.conf``
+ 
+* This could look something like this for a secure Installation you should redirect all Traffic to port 443 and use a Certificate.
```

Comment:
```suggestion
* The Nginx config should look something like the following code block.
To use `HTTPS`, you should additionally pass a certificate and redirect all traffic to port 443.
```

Reply:
For a junior user, something like `certbot` for letsencrypt should be used, it can handle all certificate related tasks.